src/Handler/RedisHandler.php

```diff
@@ -69,25 +69,25 @@ public function __construct(\Redis $redis, array $options = [])
 	 */
 	public function close()
 	{
-		$this->redis->close();
```

Comment:
i think with the destroy change we don't want to call close here?

Reply:
If I add this line back, I can no longer login. Memcache and APC close methods all return just true

Reply:
But the database does close its connection :/ given few people change the default from that I assume it’s the reference one?

Reply:
Well good luck then, I cannot find another way to fix this.